client/src/Try/Container.purs

```diff
@@ -141,12 +139,7 @@ component = H.mkComponent
           handleAction $ Compile Nothing
 
     Cache text -> H.liftEffect do
```

Comment:
Not sure if `Cache` still makes sense as the action name, maybe `Persist`?

Reply:
`EncodeInUrl` sounds good to me.

Reply:
Renamed here 6eb73be18a074c3f82065eabda4e450e981e77a7